src/jdk.incubator.code/share/classes/jdk/incubator/code/internal/CodeReflectionSymbols.java

```diff
@@ -68,12 +71,5 @@ public class CodeReflectionSymbols {
                 new MethodType(List.of(syms.stringType), opType,
                         List.nil(), syms.methodClass),
                 opParserType.tsym);
-        methodHandlesLookup = new MethodSymbol(PUBLIC | STATIC,
```

Comment:
One reason to retain this here is it is only used by the incubating `code` module, thereby minimizing changes to `java.base` and therefore reducing the maintenance when making changes independently to this area of code.

Looking more closely at its use i now realize now this declaration can be entirely removed! due to the changes in #424 we can change the code in reflect methods for structural quoting. So leave it how you wish, and I will submit a PR to remove it where ever it is located.